### DISCUSSION TITLE
Options for callback to airflow

### DISCUSSION THREAD

**Discussion by sangeetds** [apache/airflow#23956] [Q&A]
Hi everyone. 

I know we have the feature to provide the callback link/URL that Airflow DAGs can hit to provide updates after any task updates. I was curious to find out if there were any options that the Airflow DAG can provide a callback URL so that any other service can later have updates after the DAG has requested for the long-running asynchronous task.

**Comment by potiuk** [comment]
Nothing like that. There wer some ideas discussed to make in-progress task interrogation possible, but that would require implementing basically new api for every operator that runs some long-running task - even if we implement it in airflow core the actual provider's will have to be changed.

I think this would be an AIP-level change https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals so it would require quite detailed discussion in devlist, drafting a proposal and actually figuring out how to make this change applied to at least most important operators managed by the community.  If you feel like it - you can take a lead on that. AIPs can be lead by anyone, there is no need to be committter to do it.
